modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/commerce-catalogs/catalog.products.subscriptions.properties.json

```diff
@@ -0,0 +1,31 @@
+[
+	{
+		"cpDefinitionExternalReferenceCode": "RAYAP-004",
+		"cpInstanceProperties": [
+			{
+				"cpInstanceSku": "PAY IN FULL",
+				"propertyType": "UPDATE_PRICE",
+				"skuPrice": 60,
+				"skuPromoPrice": 25
+			},
+			{
+				"cpInstanceSku": "INSTALLMENT",
+				"deliveryMaxSubscriptionCycles": 0,
+				"deliverySubscriptionEnabled": false,
+				"deliverySubscriptionLength": 1,
+				"deliverySubscriptionType": "daily",
+				"maxSubscriptionCycles": 2,
+				"overrideSubscriptionInfo": true,
+				"propertyType": "CREATE_SUBSCRIPTION",
+				"subscriptionEnabled": true,
+				"subscriptionLength": 2,
+				"subscriptionType": "monthly",
+				"subscriptionTypeSettings": {
+					"monthDay": "1",
+					"monthlyMode": "0"
+				}
+			}
+		],
+		"optionKey": "raylife-ap-option"
+	}
+]
```

Comment:
Please, check if this is necessary to be created at this moment.

Reply:
Done!